contracts/PensionFundRelease.sol

```diff
@@ -109,14 +123,20 @@ contract PensionFundRelease {
 
     // release the fund
     function releaseRoots() returns (uint releasedAmount){
-        require(isReleaseApproved());
-        require(now > firstPaymentTime);
-
-        releasedAmount = getPaymentAmount();
-        if(releasedAmount > 0){
-            firtPaymentReleased = true;
+        // Confirm validators have released funds
+        if( !isReleaseApproved() || now < paymentTime){
```

Comment:
My thinking was so that it wouldn't throw an error it would return a value that can be handled but doing it with errors could work. Do you want me to switch it over?